.github/workflows/api-endpoint.yml

```diff
@@ -0,0 +1,54 @@
+name: API tests
+
+on: [push]
+
+env:
+  RUST_BACKTRACE: full
+
+jobs:
+  api:
+    name: "The API endpoints"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Install Packages"
+        run: sudo apt-get install -y libmpfr6 libmpfr-dev
+      - name: "Install Racket"
+        uses: Bogdanp/setup-racket@v1.8
+        with:
+          version: 8.5
+      - name: Install Rust compiler
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            default: true
+            override: true
+            components: rustfmt, clippy
+      - uses: actions/checkout@master
+      - name: "Install dependencies"
+        run: make install
+
+      - name: "Start server"
+        run: racket src/herbie.rkt web --port 8000 &
+      
+      - name: "Pause 4 seconds"
+        run: sleep 4s
+
+      # - name: "Get node v19"
+      #   run: curl -fsSL https://deb.nodesource.com/setup_19.x | sudo -E bash -
+      
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      # - name: "Install node"
+      #   run: sudo apt-get install -y nodejs
+
+      - name: "Print node version"
+        run: node --version
+      
+      # - name: "Test fetch"
+      #   run: node -e "fetch('https://google.com')"
+
+      - name: "Test the endpoint"
+        run: node infra/testApi.mjs
```

Comment:
Do you anticipate this file becoming more complex? So far it seems like `curl` would do just as well.

Reply:
I think it makes sense to test endpoint behavior in node rather than with command-line programs, it lets us easily unpack JSON to test whether fields appear correctly.

Reply:
Ok, fine